### PR TITLE
[dev-env] Delete file permissions

### PR DESCRIPTION
**Pull request by pschoffer**
## Description

This change:
* simplifies the lando file creation
* Uses new images for mu-plugins and dev-tools
* Combined with that above some other tweaks to make sure files will be own by the current user

## Steps to Test

1) Create env
2) Start env
3) Do some stuff
4) Restart env
5) Delete env

__no errors anywhere__

### CODE REVIEW

File: assets/dev-env.lando.template.yml.ejs [hunk 1]
```diff
@@ -116,17 +113,17 @@ services:
...
       volumes:
         - ./wordpress:/shared
 
 <% if ( muPlugins.mode == 'image' ) { %>
   mu-plugins:
     type: compose
     services:
-      image: <%= muPlugins.image %>:<%= muPlugins.tag %>
-      command: sh -c 'rsync -a /mu-plugins/ /shared/; sh /autoupdate.sh /shared'
+      image: ghcr.io/automattic/vip-container-images/mu-plugins:0.1
```
Comment (pauarge) [inline assets/dev-env.lando.template.yml.ejs:125]: mu-plugins should always be `latest`, given it's always-updating nature

File: src/lib/constants/dev-environment.js [hunk 2]
```diff
@@ -1,38 +1,22 @@
...
 	muPlugins: {
-		image: 'ghcr.io/automattic/vip-container-images/mu-plugins',
-		tag: 'latest',
+		mode: 'image',
 	},
 	clientCode: {
-		image: 'ghcr.io/automattic/vip-container-images/skeleton',
-		tag: 'latest',
+		mode: 'image',
 	},
```
Comment (pauarge) [inline src/lib/constants/dev-environment.js]: If the three are 'image', we might simplify this and specify nothing?

**Review by pauarge — CHANGES_REQUESTED** [review]
Looking good, left a couple of questions.

File: assets/dev-env.lando.template.yml.ejs (hunk 1 above)
Comment (pschoffer) [reply assets/dev-env.lando.template.yml.ejs:125]: Well not really. The mu-plugins code will be self updating but the containe itself has some other scripts and structure that I would like to start version. Here - https://github.com/Automattic/vip-container-images/pull/35 - we changed both how the docker entrypoint `run.sh` (that was unused up till now) and autoupdating works (it didn't before).

If we in the future would like to introduce some breaking change say change the folder that contains the autoupdated code from `/shared` to `/mu-plugins` we would like to NOT break all the dev-envs in the wild already expecting the folder to be `/shared`.

File: src/lib/constants/dev-environment.js (hunk 2 above)
Comment (pschoffer) [reply src/lib/constants/dev-environment.js]: Oh yeah! We dont use those anymore.

**Review by pauarge — APPROVED** [review]

**Pull request merged**